tests/base.py

```diff
@@ -361,8 +364,7 @@ def select_all_streams_and_fields(conn_id, catalogs, select_all_fields: bool = T
             non_selected_properties = []
             if not select_all_fields:
                 # get a list of all properties so that none are selected
-                non_selected_properties = schema.get('annotated-schema', {}).get(
-                    'properties', {}).keys()
+                non_selected_properties = set(schema.get('annotated-schema', {}).get('properties', {}).keys())
```

Comment:
Is there any specific reason to change the above code by converting to a set?

Reply:
No, it was changed for `event_type` field, but now as we have updated the mechanism. Reverted the change.